### ISSUE TITLE
ernie-vil: how to get same val acc using VCR finetuning model 

### ISSUE THREAD

**Issue by yanan1989** [PaddlePaddle/ERNIE#774]
Thanks for your great work. I am using the VCR finetuning model to evaluate validation set. However, I could not get a correct accuracy score as you reported in the paper? 
I got 40% val accuracy [the paper: 77%] when I used the [ERNIE-ViL-VCR-task-pretrain base] model and preprocessed data by ViLBERT.
If you have any advice I should follow up, could you please let me know?
Thank you very much!

**Comment by BinglengTang** [comment]
The [ERNIE-ViL-VCR-task-pretrain base] model is a pre-trained model. You are supposed to fine-tune the model with VCR train-split data before doing the validation.
